etc/ci-prepare.sh

```diff
@@ -37,9 +37,18 @@ echo 'Print("GAP started successfully\n");QUIT_GAP(0);' | ./gap -q -T
 # connection is refused, to work around intermittent failures
 make bootstrap-pkg-full WGET="wget -N --no-check-certificate --tries=5 --waitretry=5 --retry-connrefused"
 
+# TEMPORARY FIX : factint is not HPC-GAP compatible
+if [[ $HPCGAP = yes ]]
+then
+    echo rm -rf pkg/factint*
```

Comment:
This `echo` line is redundant, as `set -x` is active (check the travis logs if you like)